doc/source/getting-started/administration.rst

```diff
@@ -0,0 +1,161 @@
+Project approval and public release
+===================================
+Most of the projects in PyAnsys expose the functionality of ANSYS, Inc.
+products. Due to intellectual property reasons, the public release of a PyAnsys
+library must undergo through a project approval process.
+
+
+Approval process
+----------------
+The approval process is divided into three different parts:
+
+.. grid:: 3
+    
+    .. grid-item-card:: :octicon:`file-badge` Managerial
+       :link: managerial
+       :link-type: ref
+
+        Verifies that direct managers, GM/VP, CT or CTO are aware about the project and approve it.
```

Comment:
I decided to use the abbreviations here since this abstract text appears inside a card. Thus, the available space is limited. Once users click, they get redirected to the corresponding section, where all previous abbreviations are properly explained.

Hence, should we leave this as it is?